sass/layout/hero.sass

```diff
@@ -128,6 +128,13 @@
           .nav-item
             border-top-color: rgba($color-invert, 0.2)
   // Sizes
+  &.is-small
+    +tablet
+      .hero-body
+        padding-bottom: 1rem
+        padding-top: 1rem
+      .container 
+        margin: 0 auto;
```

Comment:
Ideally I would like this to function like Hello bar (see screenshot). 

If you think this should be it's own unique pattern I'm happy to build one, but I thought a modifier to the hero element (like large, full screen etc.) was more appropriate.

![hellobar](https://user-images.githubusercontent.com/454117/28549204-10463e50-709e-11e7-984e-20d7012bf96f.jpg)